addons/wasmedge/enable

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+SNAP_CHANNEL=--candidate
+PATCH_DIR=`dirname "$0"`/patch
+
+echo "Enabling wasmedge via crun OCI runtime"
+
+echo "Install/Upgrade crun-wasmedge from snapstore"
+[ ! -d "/snap/crun-wasmedge/current" ] && sudo snap install $SNAP_CHANNEL crun-wasmedge || sudo snap refresh $SNAP_CHANNEL crun-wasmedge
+sudo ln -sf /snap/crun-wasmedge/current/crun.wrapper /usr/local/bin/crun
+
+echo "Switching OCI runtime to crun with wasmedge support"
+sudo patch -p1 $SNAP_DATA/args/containerd-template.toml $PATCH_DIR/0001-add-crun-to-containerd-template.toml.patch 
+sudo systemctl restart snap.microk8s.daemon-containerd
+
+sudo touch $SNAP_DATA/var/lock/wasmedge.enabled
+echo "Enabled wasmedge via crun OCI runtime"
```

Comment:
It might be worth printing out some information here as to what happened and how would the user proceed. See for example the openebs addon [1]

[1] https://github.com/canonical/microk8s-addons/blob/main/addons/openebs/enable#L44